# This file was generated by parameters.py 

steer_angle_lim_upper: 0.7853981633974483
steer_angle_lim_lower: -0.7853981633974483
steer_effort_lim: 15.0041745
steer_velocity_lim: 4.241150082346221
steer_angle_cmd_min: 0.05
steer_mech_reduction: 189
drive_effort_lim: 3.726527
drive_velocity_lim: 21.017254852515716
drive_mech_reduction: 26.9
rover_velocity_max: 1.5762941139386786
rover_dimensions:
  d1: 0.184
  d2: 0.267
  d3: 0.267
  d4: 0.2556
  wheel_radius: 0.075
drive_noload_speed_RPM: 223
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
front_left_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: front_left_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
front_right_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: front_right_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
middle_left_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: middle_left_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
middle_right_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: middle_right_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
back_left_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: back_left_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
back_right_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: back_right_drive_joint
  pid:
    p: 2
    i: 10
    d: 0
front_left_corner_controller:
  type: effort_controllers/JointPositionController
  joint: front_left_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
front_right_corner_controller:
  type: effort_controllers/JointPositionController
  joint: front_right_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
back_left_corner_controller:
  type: effort_controllers/JointPositionController
  joint: back_left_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
back_right_corner_controller:
  type: effort_controllers/JointPositionController
  joint: back_right_corner_joint
  pid:
    p: 50
    i: 20
    d: 1
navcam_horiz_joint_controller:
  type: effort_controllers/JointPositionController
  joint: navcam_horiz_joint
  pid:
    p: 10
    i: 0
    d: 0.25
navcam_vertical_joint_controller:
  type: effort_controllers/JointPositionController
  joint: navcam_vertical_joint
  pid:
    p: 10
    i: 0
    d: 0.1
friction_coeff1_wheel: 0.6
friction_coeff2_wheel: 0.6
stiffness_wheel: 10000000
damping_wheel: 1.0
friction_coeff1_corner: 1.35
friction_coeff2_corner: 1.35
stiffness_corner: 10000000
damping_corner: 1.0
friction_coeff1_bogie: 1.35
friction_coeff2_bogie: 1.35
stiffness_bogie: 10000000
damping_bogie: 1.0
rocker_limit_upper: 0.1
rocker_limit_lower: -0.1
rover_turning_radius_min: 0.45
rover_turning_radius_max: 6.35
navcam_vertical_angle_lim_upper: 1.5707963267948966
navcam_vertical_angle_lim_lower: -1.5707963267948966
navcam_vertical_effort_lim: 1
navcam_vertical_velocity_lim: 1
navcam_horiz_angle_lim_upper: 3.141592653589793
navcam_horiz_angle_lim_lower: -3.141592653589793
navcam_horiz_effort_lim: 0.5
navcam_horiz_velocity_lim: 0.05235987755982988
navcam_baseline: 0.08
navcam_update_rate: 15
navcam_fov: 1.4486232791552935
navcam_image_width: 400
navcam_image_height: 400
navcam_clip_near: 0.01
navcam_clip_far: 300
friction_coeff1_navcam_horiz: 1.35
friction_coeff2_navcam_horiz: 1.35
stiffness_navcam_horiz: 10000000
damping_navcam_horiz: 1.0
friction_coeff1_navcam_vertical: 1.35
friction_coeff2_navcam_vertical: 1.35
stiffness_navcam_vertical: 10000000
damping_navcam_vertical: 1.0
g_sim: -1.625
